src/SimplePie.php

```diff
@@ -2115,6 +2118,27 @@ public function status_code()
         return $this->status_code;
     }
 
+    /**
+     * Get HTTP headers of the last request
+     *
+     * @return array<string, string[]>
+     */
+    public function get_http_headers(): array
```

Comment:
Not sure if we want to have this. Maybe exposing `Response` will be better.

Reply:
I think this new method is a reasonable compromise. Once we have exposed the `Response` class or implemented #190, we will be able to deprecate many methods.